EasyInventory/src/main/java/com/easyinventory/enterprise/repository/CategoryStub.java

```diff
@@ -0,0 +1,5 @@
+public class CategoryStub implements CategoryRepository {
+
+
+
+}
```

Comment:
An empty class is incomplete functionality.  Only features that meet the team's Definition of Done should be merged into master.  Otherwise, it's clutter.